vtex/faststore
# docs: update the badge documentation at faststore.dev

## What's the purpose of this pull request?
The purpose is to update the badge documentation available at [faststore.dev](https://faststore.dev/reference/ui/atoms/Badge).

## How it works? 
The update adds to the doc:

- `Use cases` section with an example of how to use the component.
- `Best practices` section: Do's and Don'ts while using the component.
- `Accessibility` section.

## How to test it?
1. Open the FastStore project in an IDE.
2. In the terminal, go to the `/faststore/docs` path.
3. Change to the `docs/update-badge-component` branch.
4. Run `yarn start`
5. With the localhost open, go to `References > Atoms > Badge` and check the doc's new sections.

### `base.store` Deploy Preview




## References
- [FastStore - Components Mapping for Documentation Makeover (WIP)](https://www.notion.so/vtexhandbook/FastStore-Components-Mapping-for-Documentation-Makeover-WIP-c7a8d435033542e0821dc8238c5cd768)
- [Components Documentation Proposal](https://www.notion.so/vtexhandbook/Components-Documentation-Proposal-38dd79a6de84469fa5062b8dee654f1f)
